backend/server.js

```diff
@@ -3526,7 +3527,10 @@ var loadData = function(callback) {
 
 async.series([
     db.init,
+    sdb.init,
+    sdb.initSemesters,
     loadData,
+    function(callback) {sdb.initCourseInfo(courseInfo, callback);},
```

Comment:
Partial application should be done as `sdb.initCourseInfo.bind(null, courseInfo)`.